asapdiscovery-data/asapdiscovery/data/fitness.py

```diff
@@ -133,8 +133,10 @@ def parse_fitness_json(target: TargetTags) -> pd.DataFrame:
             f"Specified target is not valid, must be one of: {TargetTags.get_values()}"
         )
 
-    if target == "MERS-CoV-Mpro":
-        raise NotImplementedError("MERS-CoV-Mpro fitness data not yet available.")
+    if target not in ("SARS-CoV-2-Mpro", "SARS-CoV-2-Mac1"):
```

Comment:
I have some code centralising this WIP, that I can transfer to this PR or I can do post-hoc.

Reply:
post-hoc is fine, cheers